pkg/discovery/processor/namespace_processor_test.go

```diff
@@ -81,7 +82,14 @@ func TestProcessNamespaces(t *testing.T) {
 		KubeCluster:        ks,
 	}
 
-	namespaceProcessor.ProcessNamespaces()
+	kubeNamespaceMap := namespaceProcessor.ProcessNamespaces()
```

Comment:
Does this case relate to affinity? I don't get it.

Reply:
It does, in a way. The `namespaceProcessor.ProcessNamespaces()` method was updated to return this map as a part of the affinity work. All I added was a few assertions that that map was generating the expected values.

Reply:
Make sense to me